app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render "new"
```

Comment:
1. `"Welcome to the Sample App!"`
Dùng I18n em nhé

2. Dùng sysbol thay cho string nhé
` render "new"` thành ` render :new`